### PR TITLE
Refactor mapped/unmapped sections (4.5 and 4.6)

### PR DESCRIPTION
**Pull request by MaximMoinat**
* More descriptive table caption. This is repeated for every table.
image

* Do not render the table if it is empty
image

Included a refactoring to reduce code duplication.

### CODE REVIEW

File: R/ResultsDocumentGeneration.R [hunk 1]
```diff
@@ -104,23 +104,23 @@ generateResultsDocument<- function(results, outputFolder, docTemplate="EHDEN", a
...
-      body_add_gg(plot, height=4) %>%
-      body_add_par("Figure 2. Number of records per person over time per data domain")
-
-    colnames(results$dataTablesResults$conceptsPerPerson$result) <- c("Domain", "Min", "P10", "P25", "MEDIAN", "P75", "P90", "Max")
-    doc<-doc %>% body_add_break() %>%
-      officer::body_add_par(value = "Concepts per person", style = "heading 2") %>%
-      officer::body_add_par("Table 2. Shows the number of records per person for all data domains") %>%
-      my_body_add_table(value = results$dataTablesResults$conceptsPerPerson$result, style = "EHDEN") %>%
-      officer::body_add_par(" ")
+    # plot <- recordsCountPlot(as.data.frame(results$dataTablesResults$totalRecords$result))
```
Comment (PRijnbeek) [inline R/ResultsDocumentGeneration.R:107]: Can we remove this commented section or is there a reason to keep it?

Comment (MaximMoinat) [reply R/ResultsDocumentGeneration.R]: Ah, sorry my bad. Fixed it.

**Review by PRijnbeek — COMMENTED** [review]
Maxim please check if codes < 10 are indeed removed i do not think this applies for round up queries. I think they all get set to 100 also when < 10.

**Comment by MaximMoinat** [comment]
I think that is covered in the sql (only >10 are kept, before rounding).

https://github.com/EHDEN/CdmInspection/blob/8607fbe6966953067789e38f071b3e3e53462ba6/inst/sql/sql_server/checks/mapped_drugs.sql#L12-L15

**Comment by PRijnbeek** [comment]
Yes you are righ that may need to be a parameter so mincell count parameter of the function is used? or we remove the parameter ( do not think it is used anywhere else) and fix this to 10?

**Comment by MaximMoinat** [comment]
I see, we either remove the `smallCellCount` parameter or start using it. I would say, we use it everywhere as there is a use case in lowering this in order to get a full detailed report for internal use.

**Pull request merged**